ReactWindows/ReactNative/Accessibility/ImportantForAccessibility.cs

```diff
@@ -30,5 +30,12 @@ public enum ImportantForAccessibility
         /// Hides the view and its subviews from the screen reader
         /// </summary>
         NoHideDescendants,
+
+        /// <summary>
+        /// If the accessibilityLabel is specified, it is announced; 
+        /// otherwise, the labels of its children are used. Let subviews
+        /// specify their own screen reader behavior.
+        /// </summary>
+        YesDontHideDescendants,
```

Comment:
Not 100% sure that the description is clear. It could be written in relation to what 'Yes' does. But now that I think of it how is different from auto? I would call it AutoDontHideDescendants because it does what Auto does and not Yes? We are not creating the label from children in this case?

Reply:
This one does same as Yes but does not hide children. I think this is what description says.

It's different from Auto in all respects - Auto does not say if children will be hidden or not, does not say what will be read by narrator for the control and if the control will be visible to narrator.

---
In reply to: [184246678](https://github.com/Microsoft/react-native-windows/pull/1807#discussion_r184246678) [](ancestors = 184246678)

Reply:
Then I would have the same description as for 'Yes' but with a note that it just doesn't hide the children.

---
In reply to: [184463592](https://github.com/Microsoft/react-native-windows/pull/1807#discussion_r184463592) [](ancestors = 184463592,184246678)

Reply:
Actually Yes description doesn't mention it hides the children, it will be good to add that.

---
In reply to: [184466864](https://github.com/Microsoft/react-native-windows/pull/1807#discussion_r184466864) [](ancestors = 184466864,184463592,184246678)